experiments/binned_citation_count.py

```diff
@@ -0,0 +1,308 @@
+import streamlit as st
```

Comment:
I wouldn't review the code in this file at the moment since the experiment is still a work in progress